src/components/BuildConfigCreateEditPage/BuildConfigCreateEditPage.tsx

```diff
@@ -932,15 +969,49 @@ export const BuildConfigCreateEditPage = ({ isEditPage = false }: IBuildConfigCr
         </ContentBox>
       )}
 
-      <ActionGroup>
-        <Button
-          variant="primary"
-          isDisabled={isSubmitDisabled || (selectedProduct && !selectedProductVersion) || serviceContainerScmRepositories.loading}
-          onClick={handleSubmit(isEditPage ? submitEdit : submitCreate)}
-        >
-          {isEditPage ? ButtonTitles.update : ButtonTitles.create} {EntityTitles.buildConfig}
-        </Button>
-      </ActionGroup>
+      <Form>
+        <ActionGroup>
+          <Button
+            variant="primary"
+            isDisabled={
+              isSubmitDisabled || (selectedProduct && !selectedProductVersion) || serviceContainerScmRepositories.loading
+            }
+            onClick={handleSubmit(isEditPage ? submitEdit : submitCreate)}
+          >
+            {isEditPage ? ButtonTitles.update : ButtonTitles.create} {EntityTitles.buildConfig}
+          </Button>
+
+          {buildConfigCreatingLoading && (
+            <p className="p-6 p-l-10">
+              This operation may take several minutes <LoadingSpinner isInline />
+            </p>
```

Comment:
add some styling so it is not greyed out because of the Service Container creating/updating loading state

Reply:
Discussed on PM